maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java

```diff
@@ -79,6 +84,9 @@
     private final RepositoryConnectionInfo connectionInfo;
     private boolean isSetBuild;
 
+    @Inject
```

Comment:
I thought it would inject the validator when running in container, but I could be wrong.